src/robusta/runner/telemetry.py

```diff
@@ -0,0 +1,18 @@
+import logging
+from ..model.config import Registry
+
+class Telemetry:
+    def __init__(self, registry: Registry, runner_version: str):
+        self._data = dict()
+        self._data["runner_version"] = runner_version
+
+        for k,v in registry.get_sinks().get_all().items():
```

Comment:
This is hard to find.
I would just make the Telemetry class extend BaseModel, with the telemetry attributes, and save it in the registry